frontend/templates/product-list/sections/FilterableProductsSection/useDevicePartsItemType.tsx

```diff
@@ -17,6 +18,9 @@ export function useDevicePartsItemType<T extends ProductListAttributes>(
       // `Item Type` is a single select, so just use the first value if it exists.
       return itemTypeRefinement?.refinements[0]?.value;
    }, [items]);
+   if (typeof window === 'undefined' && productList.deviceItemType) {
```

Comment:
I kinds feel like this whole routine is somewhat convoluted. I'm gonna try something.

Reply:
The pushed commit seems to not create the bookmarks weirdness and I feel it's clearer to read.